modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppStepContent.es.js

```diff
@@ -67,7 +73,35 @@ const EditAppStepContent = ({
 					/>
 				</ClayLayout.ContentCol>
 
-				{shortCutButton}
+				{shortCutButton && (
+					<ClayButton
+						displayType="secondary"
+						onClick={() => {
+							window.open(
+								workflowProcessBuilderPortletURL,
+								'_blank'
+							);
+							setShowInfoAlert(true);
+						}}
+					>
+						{Liferay.Language.get('manage-workflows')}
+						<ClayIcon className="ml-2" symbol="shortcut" />
+					</ClayButton>
+				)}
+			</ClayLayout.ContentRow>
+
+			<ClayLayout.ContentRow className="pl-4 pr-4">
+				<AlertWithLink
+					displayType="info"
+					handleClick={refetch}
+					linkMessage={` ${Liferay.Language.get('refresh')}.`}
+					message={Liferay.Language.get(
+						'refresh-the-list-if-a-newly-created-workflow-is-not-displayed'
+					)}
+					onClose={() => setShowInfoAlert(false)}
+					title={` ${Liferay.Language.get('info')}:`}
+					visible={showInfoAlert}
+				/>
```

Comment:
@naraandrade what do you think about move the alert display control implementation to outside of the stepContent component like the previous shotCurButton implementation?